cpp/util/util.cc

```diff
@@ -191,4 +199,5 @@ string ToBase64(const string& from) {
   return ret;
 }
 
+
```

Comment:
All that I like two lines between top-level declarations, I think this file can probably be dropped from your PR? :wink: